pageserver/src/walingest.rs

```diff
@@ -64,13 +60,13 @@ impl<'a, T: DatadirTimeline> WalIngest<'a, T> {
             timeline,
             checkpoint,
             checkpoint_modified: false,
-            relsize_cache: HashMap::new(),
         })
     }
 
     ///
     /// Decode a PostgreSQL WAL record and store it in the repository, in the given timeline.
     ///
+    /// This function updates `lsn` field of `DatadirModification`
```

Comment:
Look like an implementation detail, should not put in the function desc. Either put this as "//" right before the function signature or remove this line.

Reply:
Yeah, that makes sense